pkg/ccr/job.go

```diff
@@ -918,6 +959,14 @@ func (j *Job) handleDropPartition(binlog *festruct.TBinlog) error {
 	if j.SyncType == TableSync {
 		destTableName = j.Dest.Table
 	} else if j.SyncType == DBSync {
+		// use sourceTableName to judge if table is except
+		var sourceTableName string
+		sourceTableName, err = j.srcMeta.GetTableNameById(dropPartition.TableId)
```

Comment:
`GetTableNameById` is a high-cost operation, it needs to connect doris FE and execute 'SHOW TABLE'.

One possible solution is to query the table ID of the except tables before adding the job to JobManager, see `Job::FirstRun` for details.